mailpoet/assets/js/src/automation/integrations/mailpoet/steps/order_status_changed/edit/order_status.ts

```diff
@@ -0,0 +1,25 @@
+import { OrderStatusOptions } from '../../../../../types/filters';
+import { Hooks } from '../../../../../../hooks';
+
+declare global {
+  interface Window {
+    mailpoet_woo_order_status: Record<string, string>;
+  }
+}
+export const COMPLETED_ORDER_STATUS = 'wc-completed';
+export const orderStatus: OrderStatusOptions = Hooks.applyFilters(
```

Comment:
Could we improve the naming? Something like `getOrderStatusOptions` or what exactly does it return...

Reply:
done. https://github.com/mailpoet/mailpoet/blob/MAILPOET-4935-order-trigger/mailpoet/assets/js/src/automation/integrations/mailpoet/steps/order_status_changed/edit/order_status.ts#L6